.github/workflows/evolve-prod.yml

```diff
@@ -0,0 +1,23 @@
+name: Evolve Devnet ANT Registry
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  # TODO: add unit tests using bused
+  evolve:
```

Comment:
Let's add this to our build step and require tests to pass (i.e. put it all in one file)